torch_geometric/data/summary.py

```diff
@@ -56,12 +56,24 @@ def from_dataset(
                 progress bar during stats computation. If set to :obj:`None`,
                 will automatically decide whether to show a progress bar based
                 on dataset size. (default: :obj:`None`)
+            per_type_breakdown (bool, optional). If set to :obj:`True` it will
+                try to separate nodes and edges and calculate stats for each
+                type - can be usefull for i.e Hetero Dataset. If set to
+                :obj:`False` general stats for nodes and columns will be
```

Comment:
Without `only` it sounds a liitle like when it's set to `True`, general stats are not generated
```suggestion
                :obj:`False`, only the general stats for nodes and columns will be
```